src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/LogAnalyticTests.ps1

```diff
@@ -0,0 +1,44 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test Export Log Analytic Throttled Requests
+#>
+function Test-ExportLogAnalyticThrottledRequests
+{
+    $loc = "West Central US";
+    $from = Get-Date -Year 2018 -Month 2 -Day 27 -Hour 9;
+    $to = Get-Date -Year 2018 -Month 2 -Day 28 -Hour 9;
+    $sasuri = 'https://fakestore.blob.core.windows.net/mylogs/fakesas';
+    Assert-ThrowsContains { `
```

Comment:
Both of these tests should have a positive test case (can you create a new blob and container and put the logs there?)

Reply:
I cannot check in SAS Uri to GitHub because of credscan.  If I add a positive case with real SAS Uri and then remove secrets, then anyone who runs this test with record mode will fail.  Wayne already tested this manually and I also can see that this Powershell command sends a correct REST call from recorded json file.

Reply:
If you create a new storage account, do the manipulations there, and remove the storage account, you should not have a security issue.  I believe that this code is working now, but we need positive tests to ensure it continues working in the future.

Reply:
@maddieclayton  I added positive test case.  Note that storage cmdlets are not available in compute test, so I had to manually run those cmdlets.  So I commented out any manual steps in the tests.

Reply:
I think that you are able to access the cmdlets, as it is being done here: https://github.com/Azure/azure-powershell/blob/bbe5ee957de07032bcf7f9062bba91cceee62565/src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DiagnosticsExtensionTests.ps1#L42.  Am I not understanding this code correctly?  Network also uses dataplane Storage cmdlets and has no additional information in the test csproj: https://github.com/Azure/azure-powershell/blob/cd0497d8b10116ea4df648e58a9aef666cd2d79e/src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1#L481

Reply:
@maddieclayton  I update the test